src/server/mock-data/json/vergunningen.json

```diff
@@ -29,21 +29,35 @@
     {
       "caseType": "Evenement melding",
       "title": "Evenement melding",
-      "dateDecision": null,
-      "dateRequest": "2021-05-01",
-      "decision": null,
-      "documentsUrl": "/api/decosjoin/listdocuments/gAAAAABfOl8BFgweMqwmY9tcEAPAxQWJ9SBWhDTQ7AJiil0gZugQ37PC4I3f2fLEwmClmh59sYy3i4olBXM2uMWNzxrigD01Xuf7vL3DFuVp4c8SK_tj6nLLrf4QyGq1SqNESYjPTW_n",
+      "dateDecision": "2021-10-17",
```

Comment:
Kun je meerdere test vergunningen neerzetten met de belangrijkste verschillen en bv ook datums ver in de toekomst?

Dus een:
- in behandeling
- zonder decision 
etc.